### DISCUSSION TITLE
Ghidra doesn't load

### DISCUSSION THREAD

**Discussion by Luxvao** [NationalSecurityAgency/ghidra#5985] [Q&A] [answered]
I installed ghidra on arch linux and only some of it loads. I'm using openjdk 17.
![image](https://github.com/NationalSecurityAgency/ghidra/assets/95533447/8a028bec-e26c-4035-aba4-9578c0727a9e)

**Comment by ryanmkurtz** [comment]
What window manager is that?

**Reply by Luxvao** [reply]
Bspwm

**Reply by ryanmkurtz** [reply]
Can you launch Ghidra with `support/ghidraDebug` to see if any errors are reported?

**Reply by Luxvao** [reply]
No errors shown

**Comment by ryanmkurtz** [comment, marked as answer]
This is a window manager we do not test with.  I am guessing the issue is with the JDK not properly supporting Java Swing apps in this environment, based on some initial googling.  You could try this: https://www.reddit.com/r/bspwm/comments/3a5xuz/java_gui_not_fully_working/.

We are not likely to devote a lot of resources to getting this working properly (especially if it's more of a JDK issue), so depending on how badly you want it to work, we'd have to lean on you to do most of the experimentation and debugging.  Of course, if a reasonable solution was to be found, we'd consider adding it.

**Reply by Luxvao** [reply]
Exporting that environment variable fixed the issue! Thanks!
